Makefile

```diff
@@ -39,6 +39,7 @@ E2E_PARALLEL          ?= 20
 E2E_SUITE_TIMEOUT     ?= 15m
 GOTEST                ?= go test -v -p 20
 ALL_BUILD_TAGS        ?= api,cli,cron,executor,examples,corefunctional,functional,plugins
+BENCHMARK_COUNT       ?= 6
```

Comment:
This is so benchstat gives confidence intervals. Without this, it'll show the message `¹ need >= 6 samples for confidence interval at level 0.95`